server/conf/routes

```diff
@@ -83,7 +83,7 @@ GET     /admin/programs/:programId/applications/all                       contro
 GET     /admin/programs/:programId/applications/singleVersion             controllers.admin.AdminApplicationController.downloadSingleVersion(request: Request, programId: Long)
 GET     /admin/programs/:programId/applications/:applicationId            controllers.admin.AdminApplicationController.show(request: Request, programId: Long, applicationId: Long)
 GET     /admin/programs/:programId/applications/:applicationId/download   controllers.admin.AdminApplicationController.download(request: Request, programId: Long, applicationId: Long)
-GET     /admin/demographics                                               controllers.admin.AdminApplicationController.downloadDemographics()
+GET     /admin/demographics                                               controllers.admin.AdminApplicationController.downloadDemographics(request: Request)
```

Comment:
suggestion (blocking): you can specify optional query params here with `java.util.Optional[Integer]` i.e.
```suggestion
GET     /admin/demographics                                               controllers.admin.AdminApplicationController.downloadDemographics(request: Request, from_date: java.util.Optional[String], to_date: java.util.Optional[String])
```